### ISSUE TITLE
allow multiple configs

### ISSUE THREAD

**Issue by yarikoptic** [INCF/bids-validator#279]
E.g. if we have "center-wide" one, and then per-dataset specific (btw -- what would be the logical location to keep it around under?), it would be cool if I could `-c /path/center.cfg -c .validator.cfg $PWD` or something like that.  ATM second `-c` just overrides previous.

I guess it would be difficult if not impossible to "merge" configurations so most probably should be just processed in order to determine which messages (errors/warnings) could be not reported

Cheers!

**Comment by chrisfilo** [comment]
Indeed this would be a nice feature :)

**Comment by yarikoptic** [comment]
@effigies @rwblair - I found this marvel from the past: my worries of the past became my pains of the present for 

- https://github.com/OpenNeuroOrg/openneuro/pull/3141

;-)

**Comment by yarikoptic** [comment]
I wonder if adding support via simply concatenating lists of rules for each section ("ignore", "warn", ...) across provided files would allow to account for most of the use cases?  

E.g. in case of OpenNeuro users then would be able to still trigger those errors even if they were first ignored:

```shell
❯ cat $PWD/.bids-validator-config--conflicting2-deno.json
{
  "ignore": [
     "NIFTI_TOO_SMALL", { "code": "NIFTI_TOO_SMALL" },
     "SIDECAR_KEY_RECOMMENDED", { "code": "SIDECAR_KEY_RECOMMENDED" },
     "NIFTI_HEADER_UNREADABLE", { "code": "NIFTI_HEADER_UNREADABLE" },
     "TOTAL_ACQUIRED_VOLUMES_NOT_CONSISTENT", { "code": "TOTAL_ACQUIRED_VOLUMES_NOT_CONSISTENT" },
     "JSON_KEY_RECOMMENDED", { "code": "JSON_KEY_RECOMMENDED" }
    ],
  "error": [
    "JSON_KEY_RECOMMENDED", { "code": "JSON_KEY_RECOMMENDED" }
    ]
}
❯ deno run --allow-read --allow-env --reload -A https://github.com/bids-standard/bids-validator/raw/deno-build/bids-validator.js --config .bids-validator-config--conflicting2-deno.json asl001
	[ERROR] JSON_KEY_RECOMMENDED A JSON file is missing a key listed as recommended.
		GeneratedBy
		/dataset_description.json

		SourceDatasets
		/dataset_description.json

	Please visit https://neurostars.org/search?q=JSON_KEY_RECOMMENDED for existing conversations about this issue.


          Summary:                         Available Tasks:        Available Modalities:
          8 Files, 96.9 kB                                         MRI                  
          1 - Subjects 1 - Sessions                                                     

	If you have any questions, please post on https://neurostars.org/tags/bids.
``` 

sure thing that if subsequent config tries to relax condition - then it would be impossible.  For that to happen there should not be a simple concatenation but rather **keeping all those configs "action: matches" ordered and going through all of them so that the last match wins and thus decides on the level** ... and I think this would be the most consistent and robust handling where in effect subsequent config potentially overloading prior one(s) for the matching rule.
